connector_ebisumart/README.rst

```diff
@@ -0,0 +1,88 @@
+===================
+Connector Ebisumart
+===================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-qrtl%2Fsbt--custom-lightgray.png?logo=github
+    :target: https://github.com/qrtl/sbt-custom/tree/12.0/connector_ebisumart
+    :alt: qrtl/sbt-custom
+
+|badge1| |badge2| |badge3| 
+
+This module connects Odoo with Ebisumart.
+Ebisumart (https://ebisumart.com/) is a popular Japanese e-commerce platform.
+
+This module allows the synchronization of the following objects from Ebisumart to Odoo through scheduled actions:
+- Product
+- Partner
+- Sale Order
+- Purchase Order
+
+This connector supports Ebisumart API version 1.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure this module, you need to set several things in both PrestaShop and Odoo:
+
+Steps in Ebisumart
+
```

Comment:
@AungKoKoLin1997 
Why is the readme not reflecting?

Reply:
Seem like pre-commit didn't generate readme. I will rerun.

Reply:
@kanda999 After this PR https://github.com/qrtl/sbt-custom/pull/11 is merged, I will rebase to regenerate README.rst.

Reply:
I merged it.